docs/workflow-and-job.md

```diff
@@ -158,14 +158,45 @@ The most important parameter for a workflow is `runtime`. BigFlow workflows proc
 where batch means: all units of data having timestamps within a given period. The `runtime` parameter defines this period.
 
 When a workflow is deployed on Airflow, the `runtime` parameter is taken from Airflow `execution_date`. 
-It will be formatted as either `YYYY-MM-DD` or `YYYY-MM-DD hh-mm-ss`. It depends on the `Workflow` setup.
+It will be formatted as `YYYY-MM-DD hh-mm-ss`.
 
-The `Workflow` class has two additional parameters. 
+### The `schedule_interal` parameter
 
-* `schedule_interval` — Defines when a workflow should be run. It can be a cron expression or a "shortcut". 
+The `schedule_interval` parameter defines when a workflow should be run. It can be a cron expression or a "shortcut". 
 For example: `'@daily'`, `'@hourly'`, `'@once'`, `'0 0 * * 0'`.
-* `runtime_as_datetime` — Determines the `runtime` parameter format. If set as `True`, `runtime` will be `YYYY-MM-DD hh-mm-ss`, 
-otherwise `YYYY-MM-DD`.
+
+### The `start_time_expression` parameter
+
+The `start_time_expression` parameter determines the first execution date-time. Even though you provide the `start-time` parameter
+for the `build` command, you might want to start different workflows in a different time. The `start_time_expression`
+allows you to define a proper start time for each of your workflows, relative to the provided `start-time`.
+
+Let us say that you have two workflows running hourly. You want the first workflow to start in the next hour after deployment time,
```

Comment:
obstawiam, że jednak 
after/before *the* deployment time

Reply:
done